lago/providers/libvirt/cpu.py

```diff
@@ -244,12 +247,81 @@ def generate_topology(self, vcpu_num, cores=1, threads=1):
             threads=str(threads),
         )
 
+    def generate_numa(self, vcpu_num):
+        """
+        Generate guest CPU <numa> XML child
+        Configures 1, 2 or 4 vCPUs per cell.
+
+        Args:
+            vcpu_num(str): number of virtual CPUs
```

Comment:
Looks good!
Just a fix for the docstrings and we can merge.

Reply:
I fixed the docstring - I don't like it, but it is a string.

Reply:
I'm talking about:
```
+            cores(int): number of cores
+            threads(int): number of threads
```
They are not part of the function's signature.

Reply:
Fixed, thanks.